content/millennium/r4/financial/coverage.md

```diff
@@ -53,6 +53,7 @@ Search for Patient-level or Encounter-level Coverages that meet supplied query p
 _Implementation Notes_
   
 * Public Healthcare represents an insurance policy funded by a public health system such as a provincial or national health plan.
+* For Coverage search by -encounter, both Encounter.read and Coverage.read scopes are required
```

Comment:
I think Encounter.read and Coverage.read should be in quotes like seen here: https://github.com/cerner/fhir.cerner.com/blame/main/content/millennium/r4/other/binary.md#L45

Reply:
yeah, that's correct.

Reply:
<img width="871" alt="Screenshot 2020-10-08 at 3 48 55 PM" src="https://user-images.githubusercontent.com/56925992/95446351-cbd18500-097d-11eb-96fd-b6ae22a4e7b6.png">
After the change.